src/components/project/DetailButton/index.js

```diff
@@ -5,17 +5,18 @@ import CloudinaryImage from '../../CloudinaryImage';
 
 import s from './styles.module.scss';
 
-const ProjectDetailButton = ({ text, className, href, ...rest }) => (
-  <a className={classnames(s.btn, className)} href={href} {...rest}>
+const ProjectDetailButton = ({ text, className, href, child = false, ...rest }) => (
+  <a className={classnames(child ? s.btnChild : s.btn, className)} href={href} {...rest}>
```

Comment:
Maybe this is more readable:
```suggestion
  <a
    className={classnames(className, {
      [s.btnChild]: child,
      [s.btn]: !child,
    })}
    href={href}
    {...rest}
  >
```